modules/gapi/src/api/gmat.cpp

```diff
@@ -38,17 +38,45 @@ cv::GMatDesc cv::descr_of(const cv::Mat &mat)
 {
     return GMatDesc{mat.depth(), mat.channels(), {mat.cols, mat.rows}};
 }
+
 cv::GMatDesc cv::descr_of(const cv::UMat &mat)
 {
     return GMatDesc{ mat.depth(), mat.channels(),{ mat.cols, mat.rows } };
 }
+
+std::vector<cv::GMatDesc> cv::descr_of(std::vector<cv::Mat> const& vec)
```

Comment:
the rest of the code is written with `const std::vector<cv::Mat> &` order, please align since now it differs